integration/tests/params.go

```diff
@@ -77,6 +79,14 @@ func (p *Params) GenerateCommands() {
 		WithArgs("get").
 		WithTimeout(1 * time.Minute)
 
+	p.EksctlSetCmd = p.EksctlCmd.
+		WithArgs("set").
+		WithTimeout(1 * time.Minute)
+
+	p.EksctlUnsetCmd = p.EksctlCmd.
+		WithArgs("unset").
+		WithTimeout(1 * time.Minute)
+
```

Comment:
nit: you could add the `label` command to the `EksctlSet/UnsetCmd`. We've done it for other commands that only have 1 subcommand e.g. `EksctlDrainNodeGroupCmd` below:

```
	p.EksctlDrainNodeGroupCmd = p.EksctlCmd.
		WithArgs("drain", "nodegroup", "--verbose", "4").
		WithTimeout(5 * time.Minute)
```